.github/workflows/cd.yml

```diff
@@ -0,0 +1,27 @@
+name: CD
```

Comment:
I'm going to use this for the address parser API spec as well. Thx for figuring this out.